DepthImages/obstacle_detection.py

```diff
@@ -31,20 +33,20 @@ def detect(input, width, threshold):
 
     while index < len(array) - width:
         # Slope calculation
-        x1 = index
         y1 = array[index]
-        x2 = index + width
         y2 = array[index + width]
 
-        slope = (y2 - y1) / (x2 - x1)
+        slope = (y2 - y1) / width
 
-        index += width
+        index += width	# why are we doing this? it skips <width> points, is it just to be faster? operating under the assumption that any points between those we're comparing would yield similar results?
 
         # Break out of the loop if we have a non-positive slope.
-        if slope < -2:
+	# This is the case of a regular obstacle
+        if slope < -2:	# why -2? just to give a little leeway for error?
             break
         
-        if slope > 200:
+	# This is the case of a drop-off, ie stairs, curb, etc.
+        if slope > 200:	# any reason why 200?
```

Comment:
Same thing with the 200, I think this was a guess and check value for last year's group